easybuild/easyconfigs/s/SpFFT/SpFFT-0.9.8-CrayGNU-19.10-cuda-10.1.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'CMakeMake'
+
+name = 'SpFFT'
+version = '0.9.8'
+cudaversion = 10.1
+versionsuffix = '-cuda-%s' % cudaversion
+
+homepage = 'https://github.com/eth-cscs/SpFFT'
+description = """Sparse 3D FFT library with MPI, OpenMP, CUDA and ROCm support"""
+
+toolchain = {'name': 'CrayGNU', 'version': '19.10'}
+toolchainopts = {'usempi': True, 'openmp': True, 'pic': True, 'verbose': False, 'opt': True}
+
+source_urls = ['https://github.com/eth-cscs/SpFFT/archive']
+sources = ['v%(version)s.tar.gz']
+
+builddependencies = [('CMake', '3.14.5', '', True)]
+
+dependencies = [
+  ('cudatoolkit' , EXTERNAL_MODULE),
+  ('gcc', EXTERNAL_MODULE),
+  ('intel/19.0.1.144', EXTERNAL_MODULE),
```

Comment:
Suggestion: since you are adding intel to a CrayGNU recipe, maybe you could add a hint in the versionsuffix (by adding for example a `-intel`).

Reply:
Thanks. I've added intel (it is just for mkl) in the version suffix.

Reply:
You could also call use `-mkl` if you prefer. 

The last remark I have is that the file name should match the full version string (so that this recipe can be used as a dependency in another easyconfig), for example:

```
SpFFT-0.9.8-CrayGNU-19.10-cuda-10.1-intel.eb
```